org.civicrm.reqangular/src/tests/filters/angularDate/format-date_test.js

```diff
@@ -0,0 +1,91 @@
+define([
+    'common/angular',
+    'common/moment',
+    'common/angularMocks',
+    'common/filters/angularDate/format-date',
+    'common/services/angularDate/date-format',
+    'common/services/settings/hr-settings'
+], function (angular, moment) {
+    'use strict';
+
+    describe('angularDate: FormatDateFilter Unit Test', function () {
```

Comment:
Just put the name of what you are testing, it's already obvious this is a Unit Test
```js
describe('FormatDateFilter', function () {

});